diff --git a/packages/lit-html/src/lit-html.ts b/packages/lit-html/src/lit-html.ts
index cd17ee63..8922beae 100644
--- a/packages/lit-html/src/lit-html.ts
+++ b/packages/lit-html/src/lit-html.ts
@@ -1074,11 +1074,19 @@ class Template {
   // Overridden via `litHtmlPolyfillSupport` to provide platform support.
   /** @nocollapse */
   static createElement(html: TrustedHTML, _options?: RenderOptions) {
-    const el = d.createElement('template');
-    el.innerHTML = html as unknown as string;
-    return el;
+    const doc = __moz_domParser.parseFromString(
+       `<template>${html}</template>`,
+       'text/html'
+    );
+    return document.importNode(
+      doc.querySelector('template') as HTMLTemplateElement,
+      true
+    );
   }
 }
+const __moz_domParser = new globalThis.DOMParser();
+
+
 
 export interface Disconnectable {
   _$parent?: Disconnectable;
diff --git a/packages/lit/src/index.all.ts b/packages/lit/src/index.all.ts
index 1f8e5499..2d312a70 100644
--- a/packages/lit/src/index.all.ts
+++ b/packages/lit/src/index.all.ts
@@ -38,10 +38,3 @@ export {
   withStatic,
 } from './static-html.js';
 
-if (!window.litDisableBundleWarning) {
-  console.warn(
-    'Lit has been loaded from a bundle that combines all core features into ' +
-      'a single file. To reduce transfer size and parsing cost, consider ' +
-      'using the `lit` npm package directly in your project.'
-  );
-}
